examples/tts/conf/es/fastpitch_align_44100.yaml

```diff
@@ -0,0 +1,224 @@
+# This config contains the default values for training grapheme Spanish FastPitch model with aligner using
+# 44.1KHz sampling rate. If you want to train model on other dataset, you can change config values according
+# to your dataset. Most dataset-specific arguments are in the head of the config file, see below.
+
+name: FastPitch
+
+train_dataset: ???
+validation_datasets: ???
+sup_data_path: ???
+sup_data_types: ["align_prior_matrix", "pitch", "speaker_id"]
```

Comment:
`speaker_id` is not in the default English FastPitch config. Do we expect Spanish models to mainly be multi-speaker?

Reply:
I suppose fine tuning model on a single speaker could be the most common use case. But I do not have any good single speaker dataset yet (the 15 hour one I scraped from LibriVox was pretty bad). For the multi-speaker it also has the additional configs "speaker_emb_condition_prosody" and "speaker_emb_condition_aligner".

English is missing a multi speaker config example. In the German configs the 22.05kHz config is single speaker and the 44.1kHz is multi speaker, but the difference is not called out in the config name or comments.

Should I add a single speaker IPA config while adding a "_multi" suffix to these? Assuming we will need it later.

Reply:
To be more consistent, I made both configs single speaker and added a separate multi-speaker IPA config.

Reply:
Gotcha. That sounds like a good compromise. Can you add a bit of text in the header comment just to distinguish them and point out the differences for single vs. multi speaker configs?